scania/corporate-ui
# Improvement/upgrade stencil 2

**Describe pull-request**  
- Update core to stencil 2
- Use stencil store instead of Redux

**How to test**  
1. Do npm link for corporate-ui and scania-theme (https://github.com/scania/scania-theme/pull/155)
2. Check in both NPM and CDN ways of loading script
3. Check if all components work
4. Check in mobile if navigation works properly

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
